owtf/api/handlers/base.py

```diff
@@ -45,6 +45,14 @@ class BaseRequestHandler(RequestHandler):
     def set_default_headers(self):
         self.add_header("X-OWTF-Version", __version__)
         self.add_header("X-Frame-Options", "SAMEORIGIN")
+        self.add_header("X-XSS-Protection", "1; mode=block")
+        self.add_header("X-Content-Type-Options", "nosniff")
+        self.add_header("Strict-Transport-Security", "max-age=31536000; includeSubDomains")
```

Comment:
I think this will break functionality with https?

Reply:
Removed.